packages/manager/src/components/SelectRegionPanel/SelectRegionPanel.test.tsx

```diff
@@ -24,24 +50,21 @@ describe('SelectRegionPanel', () => {
       })
     );
 
-    jest.mock('react-router-dom', () => ({
-      ...jest.requireActual('react-router-dom'),
-      useLocation: () => ({
-        pathname: '/linodes/create',
-      }),
-    }));
-
     const regions = regionFactory.buildList(1, {
       id: 'id-cgk',
       label: 'Jakarta, ID',
     });
 
     const { findByText } = renderWithTheme(
-      <SelectRegionPanel
-        handleSelection={jest.fn()}
-        regions={regions}
-        selectedID="id-cgk"
-      />,
+      <MemoryRouter initialEntries={[createPath]}>
+        <Route path={createPath}>
+          <SelectRegionPanel
+            handleSelection={jest.fn()}
+            regions={regions}
+            selectedID="id-cgk"
+          />
+        </Route>
+      </MemoryRouter>,
       { flags: { dcSpecificPricing: true } }
```

Comment:
Would it be possible to use `renderWithTheme`'s `options` insted of manually wrapping with a `MemoryRouter`?

```suggestion
      <SelectRegionPanel
        handleSelection={jest.fn()}
        regions={regions}
        selectedID="id-cgk"
      />,
      {
        MemoryRouter: { initialEntries: [createPath] },
        flags: { dcSpecificPricing: true },
      }
```

Reply:
wait that's a thing?

Reply:
sure is a thing, much cleaner. Fixed, thanks for the suggestion